framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/Pages/Shared/Components/AbpPageSearchBox/AbpSearchBoxViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared.Pages.Shared.Components.AbpPageSearchBox
+{
+    public class AbpSearchBoxViewModel
```

Comment:
Delete this class! We don't want that parameter. It should be a standard text, but localized in the current language.